private/bufpkg/bufcheck/buflint/internal/buflintvalidate/util.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buflintvalidate
+
+import (
+	"reflect"
+)
+
+// validateNumberField asserts that the given numbers are valid.
+func validateNumberField[T any](m *validateField, in, notIn int, ci, lti, ltei, gti, gtei T) {
+	m.checkIns(in, notIn)
+
+	c := reflect.ValueOf(ci)
```

Comment:
Please use descriptive variable names - it's not apparent to me what `c` or `ci` are.

Reply:
resolved in [182d2e2](https://github.com/bufbuild/buf/pull/2377/commits/182d2e21541d6084e75fd345150fa990cde60d62)